data/map.txt

```diff
@@ -17790,7 +17790,7 @@ planet Dune
 planet Dustmaker
 	attributes hai
 	landscape land/badlands0
-	description `The surface of this planet, when it is not obscured by dust storms, receives enough sunlight to power not only large solar farms, but solar furnaces as well, which the Hai use for manufacturing a large variety of goods. Some of the solar farms are so large that from orbit, the light reflecting off them could almost be mistaken for lakes and small oceans. However, there is almost no water on the planet's surface except near the poles.`
+	description `The surface of this planet, when it is not obscured by dust storms, receives enough sunlight to power not only large solar farms, but solar furnaces as well, which the Hai use for manufacturing a large variety of goods. Some of the solar farms are so large that from orbit the light reflecting off them could be mistaken for lakes and small oceans. However, there is almost no water on the planet's surface except near the poles.`
```

Comment:
I didn't pull it from a book, that's just how it seems like it should be written based on speech patterns. Googling it, it seems to be an "aside" to me. The sentence flows fine without it, but it provides some additional context. If I were speaking correctly, I'd pause there at those commas. 

http://books.heinemann.com/shared/onlineresources/E00629/LCardsP-3.pdf